.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: docker
+    directory: "/"
+    schedule:
+      interval: daily
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
would add a limit of prs per day

```
open-pull-requests-limit: 10
```

Reply:
@KeisukeYamashita can you update this, please? I'll approve your PR after. thanks